src/base/Quantity.hh

```diff
@@ -48,20 +53,28 @@ struct UnitlessQuantity
  * A relativistic equation that operates on these quantities can do so without
  * unnecessary floating point operations involving the speed of light:
  * \code
-   real_type eval = energy.value(); // Natural units
-   MevMomentum momentum{std::sqrt(eval * eval + 2 * mass.value() * eval)};
+   real_type eval = qvalue<MevEnergy>(energy); // Natural units
+   MevMomentum momentum{std::sqrt(eval * eval
+                                  + 2 * qvalue<MevMass>(mass) * eval)};
    \endcode
  * The resulting quantity can be converted to the native Celeritas unit system
- * with a `unit_cast`, which multiplies in the constant value of
+ * with `from_quantity`, which multiplies in the constant value of
  * ElMomentumUnit:
  * \code
- * real_type mom = unit_cast(momentum);
+ * real_type mom = from_quantity(momentum);
  * \endcode
  *
+ * When using a Quantity from another part of the code, e.g. an imported unit
+ * system, use the \c quantity free function rather than \c .value() in order
```

Comment:
```suggestion
 * system, use the \c from_quantity free function rather than \c .value() in order
```
Is that the intent?

Reply:
Yeah, generally I think two separate pieces of code should be using `qvalue` (or whatever it gets renamed to) to guarantee the coupling units are correct. However, I think we'll have use cases when all the types are "local" and known so there's no extra need for type safety:
```c++
void Particle::do_it_accelerate_me(Energy e)
{
    this->energy = Energy{this->energy.value() + e.value()};
}
```